api/types.go

```diff
@@ -693,27 +693,37 @@ type AssociationTarget struct {
 
 func toSSMAssociationDescription(rawDesc *ssm.DescribeAssociationOutput) *AssociationDescription {
 	const dateLayout = "2006-01-02 15:04:05 +0000"
+	formatDate := func(t *time.Time) string {
```

Comment:
We already have `timeFormat` and `tzTimeFormat` functions defined, you should use one of those

Reply:
In ruby response, for GET SSMAssociation,  we are getting the time format as above ex: "2022-03-14 16:19:34 +0000". Please let me know if you want me to use `timeFormat` or `tzTimeFormat`